src/rtos/semphr.c

```diff
@@ -202,6 +202,10 @@ uint8_t mutex_take(mutex_t mutex, uint32_t timeout) {
 	return xQueueSemaphoreTake( ( mutex ), ( timeout ) );
 }
 
+void mutex_delete(mutex_t mutex){
+  queue_delete((queue_t)(mutex));
```

Comment:
sem_delete is what freeRTOS has doc'd on their website. It's the same code, but can you use that instead?

Reply:
This is the same way sem_delete is implemented on line 522 of this file

Reply:
I know. Using sem_delete doesn't break abstraction barriers though and is what free rtos recommends.

Reply:
Are you saying to use sem_delete from this file or freeRTOS's vSemaphoreDelete? I think you might have been saying the former while I assumed you meant the later.

Reply:
This one: https://github.com/purduesigbots/pros/blob/7862fc52e2f351cb7977305c4595bfb50c3a87fc/src/rtos/semphr.c#L518

Reply:
It references the other function now